statsmodels/graphics/tsaplots.py

```diff
@@ -48,6 +52,9 @@ def _plot_corr(ax, title, acf_x, confint, lags, irregular, use_vlines,
             lags = lags[1:]
             confint = confint[1:]
             acf_x = acf_x[1:]
+        lags = lags.astype(np.float)
+        lags[0] -= 0.5
+        lags[-1] += 0.5
```

Comment:
what's the reason for this?
Doesn't this keep the fill_between too far away from the plot left and right edges?

Reply:
Nope, it fills right to the point and so 1/2 the dot is outside the CI on the edges.

This is from master:

![image](https://user-images.githubusercontent.com/5585221/44416829-ee20a280-a56b-11e8-8dfa-5482753c0649.png)